vault_core/src/vault_staker_withdrawal_ticket.rs

```diff
@@ -104,6 +103,13 @@ impl VaultStakerWithdrawalTicket {
         ])
     }
 
+    /// Returns the seeds for the PDA used for signing
+    pub fn signing_seeds(&self, vault: &Pubkey) -> Vec<Vec<u8>> {
```

Comment:
You don't need to pass in the vault here - just use self.vault

Reply:
Oops, thanks! removed vault pubkey.